.github/workflows/createTestList.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import os
+import ast
+
+def main():
+    
+    sanity_format = "sanity.{}"
+    formattedTests = []
+    
+    targetNames = ast.literal_eval(os.getenv('TARGET_LIST'))
```

Comment:
Instead of using the environment variable I would suggest parse the program arguments.